src/translate.spec.js

```diff
@@ -60,4 +63,46 @@ describe('translate enhancer', () => {
             expect(TranslatedComponent.displayName).toBe(translatedDefaultName);
         });
     });
+
+    it('should not re-render component on every non-related dispatch call', async () => {
+        let pChanged = false;
+        let dispatched = true;
```

Comment:
I just added it to be sure that the action was dispatched. It ensures that if the react internals introduce any delay in life cycle methods, the test will fail.